index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="source/style.css" />
+        <meta name=viewport content="width=device-width, initial-scale=1">
+        </head>
+<body>
+    <div class="logo">My Logo</div>
+    <div class="box container"> 
+        <nav class="topnav">
+            <a href="#Portoflio">Portoflio</a>
+            <a href="#About">About</a>
+            <a href="#contact">Contact</a>
+         </nav>
+    </div>
+    <div class="box flex-container">
+        <div class="text">I’m Karam Glad to see you!</div>
+        <div class="text1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div><br>
+        <div class="text3">LET’S CONNECT</div>
+     </div>
+     <div class="box row">
+        <div class="ellipse3"><img src="source/Ellipse3.png" alt=""></div>
+        <div class="ellipse11"><img src="source/Ellipse11.png" alt=""></div>
+        <div class="ellipse4"><img src="source/Ellipse4.png" alt=""></div>
+        <div class="Rectangle6"><img src="source/Rectangle6.png" alt=""></div>
+        <div class="Rectangle5"><img src="source/Rectangle5.png" alt=""></div>
+        <div class="Rectangle7"><img src="source/Rectangle7.png" alt=""></div>
+        <div class="ellipse5"><img src="source/Ellipse5.png" alt=""></div>
+        <div class="ellipse12"><img src="source/Ellipse12.png" alt=""></div>
+        <div class="ellipse13"><img src="source/Ellipse13.png" alt=""></div>
+        <div class="Rectangle4"><img src="source/Rectangle4.png" alt=""></div>
+    </div> 
```

Comment:
- [ ] The section has the proper background image(s), decorations are always background images, not `<img>` tags. Please kindly fix that.